### PR TITLE
[CELEBORN-1788] Add role and roleBinding helm charts

### PR DESCRIPTION
**Pull request by zhaohehuhu**
### What changes were proposed in this pull request?

as title

### Why are the changes needed?

 help service account control what permissions and resources a pod has access to.

### Does this PR introduce _any_ user-facing change?

no

### How was this patch tested?

test the template rendering by helm template command line

### CODE REVIEW

**Comment by zhaohehuhu** [comment]
@FMX plz help review. Thanks

**Review by FMX — COMMENTED** [review]
Is there any approach to adding UTs to this PR?

**Comment by zhaohehuhu** [comment]
> Is there any approach to adding UTs to this PR?

Okay. I'll try to add some UTs

File: charts/celeborn/ci/values.yaml [hunk 1]
```diff
@@ -140,3 +140,12 @@ serviceAccount:
...
   name: default
+
+rbac:
+  create: false
+  roleName: default
+  roleBindingName: default
+  rules:
+    - apiGroups: [""]
+      resources: ["pods"]
+      verbs: ["create", "list", "delete"]
```
[the remark this answers is not in the document]
Comment (zhaohehuhu) [reply charts/celeborn/ci/values.yaml:151]: The UT may be failed if so. Let me try it again.

[the remark this answers is not in the document]
Comment (FMX) [reply charts/celeborn/ci/values.yaml:151]: OK, the lint check go first.

[the remark this answers is not in the document]
Comment (zhaohehuhu) [reply charts/celeborn/ci/values.yaml:151]: UT failed as expected

**Review by FMX — APPROVED** [review]
LGTM. Thanks. Merged into main(v0.6.0).

**Comment by ChenYi015** [comment]
Hi @zhaohehuhu , I am wondering why we need to grant RBAC permissions to Celeborn pods? I think Celeborn cluster can work as expected with no extra permissions granted since they do not need to interact with K8s api server.

**Pull request closed without merging**